src/utilities/findUserWithFullName/findUserWithFullName.js

```diff
@@ -0,0 +1,7 @@
+export function findUserWithFullName(usersArray, fullName) {
+  const firstName = fullName.split(' ')[0];
+  const lastName = fullName.split(' ')[1];
+  return usersArray.find((user) => {
+    return user.firstName === firstName && user.lastName === lastName;
```

Comment:
A better solution would be to combine the `user.firstName` and `user.lastName` and compare it with the `fullName`